src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  const baseCost = days * dailyRate;
+  let discount = 0;
+
+  if (days >= 7) {
+    discount = 50;
+  } else if (days >= 3) {
+    discount = 20;
```

Comment:
Please replace the hardcoded value '3' with a named constant to make it clear what this value represents, as per the checklist item 2.